rviz2_plugin/CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE "RVIZ_DEFAULT_PLUGINS_BUILDIN
 pluginlib_export_plugin_description_file(rviz_common plugin_description.xml)
 
 ament_export_include_directories(include)
-ament_export_interfaces(${PROJECT_NAME} HAS_LIBRARY_TARGET)
+ament_export_targets(${PROJECT_NAME} HAS_LIBRARY_TARGET)
```

Comment:
Can we make this eloquent compatible too just like in https://github.com/osrf/rmf_demos/pull/126 ?

Reply:
417f6bb